generator/Generator.scala

```diff
@@ -1246,7 +1247,7 @@ def generateTestClasses(): Unit = {
 
             @$test
             public void shouldRecognizeArbitraryError() {
-                final Arbitrary<?> arbitrary = n -> { throw new RuntimeException("woops"); };
+                final Arbitrary<?> arbitrary = n -> { throw new RuntimeException("$woops"); };
```

Comment:
Shouldn't `"$woops"` actually be `s"$woops"`.
The same happens few times below too.